openstax/api.py

```diff
@@ -1,229 +1,20 @@
-from wagtail.api.v2.endpoints import PagesAPIEndpoint, BaseAPIEndpoint
+from wagtail.api.v2.views import PagesAPIViewSet
+from wagtail.images.api.v2.views import ImagesAPIViewSet
+from wagtail.documents.api.v2.views import DocumentsAPIViewSet
 from wagtail.api.v2.router import WagtailAPIRouter
-from wagtail.images.api.v2.endpoints import ImagesAPIEndpoint
-from wagtail.documents.api.v2.endpoints import DocumentsAPIEndpoint
-from wagtail.core.models import Page
+#from wagtail.core.models import Page
+from wagtail.admin.api.views import PagesAdminAPIViewSet
 
-## TODO: This can be removed when Wagtail fixes admin API
-from collections import OrderedDict
-from wagtail.admin.api.endpoints import PagesAdminAPIEndpoint
-from wagtail.api.v2.filters import (
-    ChildOfFilter, DescendantOfFilter, FieldsFilter, OrderingFilter,
-    SearchFilter, ForExplorerFilter)
-from wagtail.admin.api.serializers import AdminPageSerializer
-from wagtail.admin.api.filters import HasChildrenFilter
-from wagtail.api.v2.utils import BadRequestError, filter_page_type, page_models_from_string
-from wagtail.core import hooks
-from wagtail.core.models import Page, UserPagePermissionsProxy
-## TODO: END
-
-from oxauth.views import get_user_data
-from oxauth.functions import get_user_info
-
-from django.utils import timezone
-
-from .functions import remove_locked_links_listing, remove_locked_links_detail
-
-from flags.state import flag_state
-
-import json
-
-
-# Create the router. "wagtailapi" is the URL namespace
+# # Create the router. "wagtailapi" is the URL namespace
 api_router = WagtailAPIRouter('wagtailapi')
```

Comment:
These don't need to be here if you're removing the ForExplorerFilter below. There should also be some changed to remove these from openstax/urls.py.

Reply:
Fixed as we discussed